exe/kubernetes-deploy-restart

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+require 'kubernetes-deploy'
+require 'kubernetes-deploy/restart_task'
+
+raw_deployments = nil
+ARGV.options do |opts|
+  opts.on("--deployments=LIST") { |v| raw_deployments = v }
+  opts.parse!
+end
+
+if raw_deployments.nil?
+  puts "Failed: specify at least one deployment to restart with --deployments flag"
+  exit 1
+end
+
+logger = Logger.new(STDERR)
+logger.level = Logger::INFO
+logger.formatter = ->(severity, _datetime, _progname, msg) do
+  "[#{severity}] #{msg}\n"
+end
+
+restart = KubernetesDeploy::RestartTask.new(namespace: ARGV[0], context: ARGV[1], logger: logger)
```

Comment:
Looks like if the context is invalid, we'll raise `"#{context} context must be configured in your KUBECONFIG (#{ENV['KUBECONFIG']}). "` in the client builder, but what does the failure look like when the namespace arg is either missing or invalid?